sycl/doc/extensions/supported/sycl_ext_oneapi_sub_group_mask.asciidoc

```diff
@@ -100,6 +113,37 @@ The mask is defined such that the least significant bit (LSB) corresponds to
 the work-item with id 0, and the most significant bit (MSB) corresponds to the
 work-item with the id `max_local_range()-1`.
 
+|===
+|Constructor|Description
+
+|`sub_group_mask()`
+|Constructs a group mask with all bits set to 0. Size of a group mask
+corresponds to max local range of a sub-group which work-item belongs to.
+
+|`sub_group_mask(unsigned long long val)`
+|Constructs a group mask with the first `N` bit positions to the
+corresponding bit values in _val_. `N` is a size of a group mask and it
+corresponds to max local range of a sub-group which work-item belongs to. If
+size of a group mask is bigger than number of bits in the value representation
+of `unsigned long long`, the remaining positions are initialized to zero.
+
+|`template <typename T> sub_group_mask(const T& &val)`
+|Constructs a group mask with the first `N` bit positions to the
+corresponding bit values in _val_. `T` must be a SYCL `marray` of integral
+types. `N` is a size of a group mask and it corresponds to max local range of a
```

Comment:
> `T` must be a SYCL `marray` of integral types. 
```suggestion
|`template <typename T, std::size_t N> sub_group_mask(const sycl::marray<T, N> &val)`
|Constructs a group mask with the first `N` bit positions to the
corresponding bit values in _val_. `T` must be an integral type.
`N` is a size of a group mask and it corresponds to max local range of a
```

Reply:
Applied in 366c3ea5f81dbaec0d4429f593f6e74b72eec266, but with a slight change: `N` in the description is not the same as number of elements in `val`, but it is a max size of the sub-group current work-item belongs to.